docs/docs/start/quick-start.mdx

```diff
@@ -8,7 +8,7 @@ aliases:
   - "/docs/start/"
 ---
 
-import { FluidBundleLoader} from "@site/src/components/fluidBundleLoader";
+import { MockDiceRollerSample } from "@site/src/components/mockDiceRoller";
```

Comment:
This could be done outside of this PR. It's used by the homepage but is also used in our other docs. I'll push these changes directly to the base branch to reduce the PR scope here.